ui/react-components/components/AppointmentEditor/AppointmentEditor.jsx

```diff
@@ -71,13 +78,14 @@ const AppointmentEditor = props => {
         notes: undefined,
         startDateType: undefined,
         endDateType: undefined,
-        recurrenceType: undefined,
+        recurrenceType: dayRecurrenceType,
         occurrences: undefined,
         period: undefined,
-        weekDays: undefined
+        weekDays: undefined,
     };
 
     const [appointmentDetails, setAppointmentDetails] = useState(initialAppointmentState);
```

Comment:
can appointmentDetails be just called Appointment? Why is it called 'appointmentDetails'??

Reply:
We will handle this as a separate tech debt card.